libdnf/base/log_event.cpp

```diff
@@ -126,6 +126,8 @@ std::string LogEvent::to_string(
                 throw std::invalid_argument("Missing SolverProblems to convert event into string");
             }
             return ret.append(solver_problems->to_string());
+        case GoalProblem::WRITE_DEBUG:
+            return ret.append(utils::sformat(_("Debug data written to \"{}\""), *additional_data->begin()));
```

Comment:
This is just wrong, it should not be a part of the goal event log.

Reply:
May be but this solution is not harmful. The information can be easily used by Daemon. You have to also consider that the process of writing debugdata is completely driven by goal resolve according to configuration therefore the report must be also generated indirectly. The report contains essential information (path) therefore we have to provide it in for that can be easily parsed.

Reply:
If you need this mechanism to get the information out, it is a hack, and hence it is definitely harmful. And the fact that it's an easy solution is no justification for poor design.

I've already said before the directory for the debugdata most likely shouldn't come from the config file (which should save you the need to propagate out this information), but regardless of that, we're writing new code and it should be designed properly, so there should be only records that describe the transaction in the goal event log, not these arbitrary messages.